src/util/schema-translator.ts

```diff
@@ -0,0 +1,92 @@
+import { IFrontendEngineData } from "@lifesg/web-frontend-engine/components";
```

Comment:
FEE does not need sub-directory imports because it eagerly loads all components

```suggestion
import { IFrontendEngineData } from "@lifesg/web-frontend-engine";
```